catroid/src/main/java/org/catrobat/catroid/stage/StageLifeCycleController.java

```diff
@@ -112,6 +113,8 @@ static void stageCreate(final StageActivity stageActivity) {
 		stageActivity.stageAudioFocus = new StageAudioFocus(stageActivity);
 		stageActivity.stageResourceHolder = new StageResourceHolder(stageActivity);
 
+		StageActivity.webConnectionHolder = new WebConnectionHolder();
```

Comment:
please dont separate the initialization of a static field into another class...

and probably the WebConnectionHolder would be placed best in the stageListener, and not static.